x-pack/plugins/aiops/public/application/utils/url_state.ts

```diff
@@ -40,6 +39,20 @@ export const getDefaultAiOpsListState = (
   ...overrides,
 });
 
-export const isFullAiOpsListState = (arg: unknown): arg is AiOpsFullIndexBasedAppState => {
-  return isPopulatedObject(arg, Object.keys(getDefaultAiOpsListState()));
+export interface LogCategorizationPageUrlState {
```

Comment:
Did you manage to find out if it's possible to disable auto refresh on the page? It's good that we persist the field selection, but we lose the results when the page refreshes (inside ML).

Reply:
I could not find a way to do it. you can disable some parts of the time picker, but not the refresh checkbox.